spec/controllers/redirect/contact_controller_spec.rb

```diff
@@ -5,13 +5,14 @@
     stub_analytics
   end
 
+  subject(:response) { get :show, params: }
```

Comment:
`response` is a magic/reserved existing word in controller specs. I'd rename this to `action`

Then you can do

```
response = action
```
```suggestion
  subject(:action) { get :show, params: }
```